add section for server side (#5661)
segmentio/segment-docs: src/connections/destinations/catalog/batch/index.md

[SEGAnalytics setupWithConfiguration:config];
```

> ## Server Side

Reviewer: hey @tcgilbert - looks like the iOS and Android repositories on Batch's end are archived with a message saying "This integraion has been deprecated and does not work with recent Segment SDKs. Please contact us for more info on how to connect Segment and Batch depending on your usecase."

Can we take the links to those repos out of the docs? Do they want to update the docs with more information on how our mutual customers can get this integration set up, or is that what the section below does?

> Transmitting server-side data from Segment to Batch can be accomplished with the use of a [destination function](/docs/connections/functions/destination-functions/){:target="_blank"}. Follow the steps outlined [in these docs](https://help.batch.com/en/articles/2208243-how-to-connect-batch-to-segment){:target="_blank"} for a smooth integration.

Reviewer: ```suggestion
You can transmit server-side data from Segment to Batch with a [destination function](/docs/connections/functions/destination-functions/). Follow the steps outlined [in Batch's documentation](https://help.batch.com/en/articles/2208243-how-to-connect-batch-to-segment){:target="_blank"} for a smooth integration.
```
